R/data_type.R

```diff
@@ -234,3 +239,35 @@ split_raw_data <- function(df,
     x[, selected_columns, with = FALSE]
   }, df_list)
 }
+
+#' @keywords internal
+#'
+unify_combination_data <- function(df, cl, drug_ids) {
```

Comment:
@bczech could you add some test for this function?

Reply:
Done